src/pages/request-access.tsx

```diff
@@ -30,6 +58,8 @@ export default function RequestDemo(): JSX.Element {
 
   const formRef = React.useRef(null);
 
+  const formId = i18n.currentLocale === "ja" ? 3688 : 2753;
```

Comment:
Might it be better to use the `useIsJapanese` helper here? Not a blocker to merging.